spec/requests/api/v1/merchants/merchant_business_requests_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+RSpec.describe 'Merchant Business API', type: :request do
+  before(:each) do
+    Merchant.destroy_all
+    Item.destroy_all
+    Invoice.destroy_all
+    InvoiceItem.destroy_all
+    Transaction.destroy_all
+    Customer.destroy_all
```

Comment:
I wonder if there is a better way to ensure all records are destroy before each test.